scripts/generate.sh

```diff
@@ -25,16 +25,29 @@ generate_stringer() {
 #
 # $1: plugin
 # $2: file
+# $3: other options
```

Comment:
It's a weird concept here - someone using this function has to really know what they are trying to do. A full explanation here is probably out of scope I'd say, we'd want a lot more documentation in this file if we're doing things like that :)